components/generic/RightsStatement.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <span data-qa="rights statement">
+    <span
+      v-for="icon in rights.iconClass"
+      :key="icon"
+      :class="icon"
+      class="license"
+    />
+    {{ rights.name }}
+  </span>
+</template>
+
+<script>
+  export default {
+    props: {
+      rightsStatementUrl: {
+        type: String,
+        default: ''
+      }
+    },
+    computed: {
+      rights: function () {
```

Comment:
Wonder if this could be a little more descriptive?

also, we can write the function as `rights () {}`. Will keep it more consistent throughout.

Reply:
I have updated the name of the function and I agree to keep it consistent, but we are using nameOfFunction: function () {} for almost every computed property functions.